### PR TITLE
Update k8scloudconfig with the changes needed

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/2550

### CODE REVIEW

**Comment by pipo02mix** [comment]
Tested in gauss. PTAL

**Comment by rossf7** [comment]
Looks like the component versions and changelog for the version bundle still need to be updated.

File: service/version_bundles.go [hunk 1]
```diff
@@ -383,6 +383,11 @@ func NewVersionBundles() []versionbundle.Bundle {
 					Description: "Change default etcd data dir to /var/lib/etcd.",
 					Kind:        versionbundle.KindChanged,
 				},
+				{
+					Component:   "aws-operator",
+					Description: "Allow persistent volumes be automatically extended when the claims are changed.",
+					Kind:        versionbundle.KindAdded,
+				},
```
Comment (pipo02mix) [inline service/version_bundles.go]: @rossf7 it is not here?

**Comment by pipo02mix** [comment]
> Looks like the component versions and changelog for the version bundle still need to be updated.
Which component version?
it is k8scloudconfig update after adding a tag to the api server manifest

File: Gopkg.lock [hunk 2]
```diff
@@ -136,12 +136,12 @@
   branch = "master"
   name = "github.com/giantswarm/k8scloudconfig"
   packages = [
-    "v_2_0_0",
```
Comment (fgimenez) [inline Gopkg.lock]: I'm curious, where does this change come from?

**Comment by rossf7** [comment]
@pipo02mix I checked the k8scloudconfig changelog in vendor. It says calico and etcd were updated but the component versions in the bundle are the old ones.

> Updated calico to 3.0.2.
> Updated etcd to 3.3.1.

File: Gopkg.lock (hunk 2 above)
Comment (rossf7) [reply Gopkg.lock]: Yesterday I changed legacy `awsconfig/v1` from `v_2_0_0` to `v_2_0_2` for the kube-proxy fix. Not sure why this didn't change then but I think this is OK.

See https://github.com/giantswarm/aws-operator/pull/766/files

**Pull request merged**